test/gst-msdk/encode/av1.py

```diff
@@ -0,0 +1,58 @@
+###
+### Copyright (C) 2019-2021 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.gstreamer.msdk.util import *
+from ....lib.gstreamer.msdk.encoder import EncoderTest
+
+spec = load_test_spec("av1", "encode", "8bit")
+
+@slash.requires(*have_gst_element("msdkav1enc"))
+@slash.requires(*have_gst_element("msdkav1dec"))
+class AV1EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec         = "av1-8",
+      gstencoder    = "msdkav1enc",
+      gstdecoder    = "msdkav1dec",
+      gstmediatype  = "video/x-av1",
+      gstmuxer      = "matroskamux",
+      gstdemuxer    = "matroskademux",      
+    )
+
+  def get_file_ext(self):
+    return "ivf"
```

Comment:
Seems we should use `".webm"` file extension since we are using the `matroskamux`.

Reply:
or remove the `gstmuxer/gstdemuxer` setting and encode directly to `ivf` (NOTE: needs parser)?